src/core/sprites/webgl/SpriteRenderer.js

```diff
@@ -452,13 +452,7 @@ export default class SpriteRenderer extends ObjectRenderer
 
         super.destroy();
 
-        for (let i = 0; i < this.shaders.length; i++)
-        {
-            if (this.shaders[i])
-            {
-                this.shaders[i].destroy();
-            }
-        }
+        this.shader.destroy();
```

Comment:
check for existence, construction and immediate destruction without context creation would make this error out.